crypto/async/arch/async_win.h

```diff
@@ -27,7 +27,8 @@ typedef struct async_fibre_st {
 # define async_fibre_swapcontext(o,n,r) \
         (SwitchToFiber((n)->fibre), 1)
 # define async_fibre_makecontext(c) \
-        ((c)->fibre = CreateFiber(0, async_start_func_win, 0))
+        ((c)->fibre = CreateFiberEx(0, 0, FIBER_FLAG_FLOAT_SWITCH, \
+                async_start_func_win, 0))
```

Comment:
Please line up the parameters with the start of the parameters on the previous line